gui/packages/desktop/src/renderer/app.js

```diff
@@ -360,18 +360,78 @@ export default class AppRenderer {
     actions.settings.updateAutoConnect(autoConnect);
   }
 
+  async _getAppComponentsVersions() {
+    const daemonVersion = await this._daemonRpc.getCurrentVersion();
+    const guiVersion = remote.app.getVersion().replace('.0', '');
+    return {
+      daemon: daemonVersion,
+      gui: guiVersion,
+      isConsistent: daemonVersion === guiVersion,
+    };
+  }
+
   async _fetchCurrentVersion() {
     const actions = this._reduxActions;
-    const versionFromDaemon = await this._daemonRpc.getCurrentVersion();
-    const versionFromGui = remote.app.getVersion().replace('.0', '');
+    const versions = await this._getAppComponentsVersions();
+
+    // notify user about inconsistent version
+    if (process.env.NODE_ENV !== 'development' && !versions.isConsistent) {
+      this._notificationController.notifyInconsistentVersion();
+    }
 
-    actions.version.updateVersion(versionFromDaemon, versionFromDaemon === versionFromGui);
+    actions.version.updateVersion(versions.gui, versions.isConsistent);
```

Comment:
That was a strange omission that GUI displayed the daemon's version but not its own 😆 Although the versions between GUI and daemon should never differ. I am pretty sure `-dev` version will be missed from settings. 😢